apps/frontend/src/app/5_pages/BitocracyPage/components/NewProposalForm/components/ProposalOverview/ProposalOverview.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { FC, useMemo, useState } from 'react';
+
+import { t } from 'i18next';
+
+import {
+  Button,
+  ButtonStyle,
+  FormGroup,
+  Icon,
+  IconNames,
+  Input,
+  InputSize,
+} from '@sovryn/ui';
+
+import { translations } from '../../../../../../../locales/i18n';
+import {
+  ProposalOverviewData,
+  ProposalType,
+} from '../../NewProposalForm.types';
+
+export type ProposalOverviewProps = {
+  value: ProposalOverviewData;
+  onChange: (value: ProposalOverviewData) => void;
+  proposalType: ProposalType;
+  onBack: () => void;
+};
+
+export const ProposalOverview: FC<ProposalOverviewProps> = ({
+  value,
+  onChange,
+  proposalType,
+  onBack,
+}) => {
+  const [form, setForm] = useState(value);
+
+  const isSubmitDisabled = useMemo(
```

Comment:
The validation is incorrect - you should check if all the form inputs have some value but this condition means it's valid if ANY of them has some value.